utils/sl/check_script.py

```diff
@@ -33,6 +33,7 @@
 from distutils.spawn import find_executable
 #pylint: enable=import-error
 #pylint: enable=no-name-in-module
+#pylint: disable=consider-using-with
```

Comment:
Do you know why we have some directives here instead of https://github.com/daos-stack/daos/blob/master/utils/sl/pylint3.rc? Is `pylint3.rc` for all  python and not just functional tests?

Reply:
pylint3.rc is for all python code, these entries are just disabling it for this file.